docs/src/main/paradox/discovery.md

```diff
@@ -176,6 +176,43 @@ spec:
           protocol: TCP
 ```
 
+## Discovery Method: AWS EC2 Discovery
+
+If you're an AWS user, you can use tags to simply mark the instances that belong to the same cluster. Use a tag that
+has "service" as the key and set the value to be the name of your service. Note that this implementation is adequate
```

Comment:
Could it be useful to show how to add such tag using the CLI or the UI a screenshot?

Reply:
done :-)